app/src/features/settings/components/BillingTeam/components/BillingDetails/MyBillingTeamDetails/components/BillingTeamMembers/index.tsx

```diff
@@ -101,61 +101,66 @@ export const BillingTeamMembers: React.FC<Props> = ({ openDrawer }) => {
+                <IoMdCloseCircleOutline fontSize={16} className="mr-8" />
+                Remove
+              </Row>
+            </Popconfirm>
+          ),
+        };
         return (
           <Row
             justify="end"
             align="middle"
             gutter={8}
-            className={`w-full  ${loadingRows.includes(record.id) ? "loading-cell" : ""}`}
+            className={`w-full ${loadingRows.includes(record.id) ? "loading-cell" : ""}`}
           >
-            <Col>
-              <Popconfirm
-                title="Are you sure you want to remove this user from the billing team?"
-                onConfirm={() => {
-                  setLoadingRows([...loadingRows, record.id]);
-                  trackBillingTeamActionClicked("remove_member");
-                  removeMemberFromBillingTeam(billingId, record.id)
-                    .then(() => {
-                      toast.success("User removed from the billing team");
-                      trackBillingTeamMemberRemoved(record.email, billingId);
-                    })
-                    .catch(() => {
-                      toast.error("Error while removing user");
-                    })
-                    .finally(() => {
-                      setLoadingRows(loadingRows.filter((row) => row !== record.id));
-                    });
-                }}
-                okText="Confirm"
-                cancelText="Cancel"
-              >
-                <RQButton
-                  type="text"
-                  icon={<IoMdCloseCircleOutline fontSize={14} />}
-                  className="remove-member-btn"
-                  disabled={!isUserAdmin}
-                >
-                  Remove
-                </RQButton>
-              </Popconfirm>
-            </Col>
             <Col>
               <Dropdown
                 menu={{
-                  items: items.map((item) => ({ ...item, disabled: item.key === record.role })),
+                  items: [...items, removeAction].map((item) => ({ ...item, disabled: item.key === record.role })),
                   onClick: ({ key }) => {
-                    setLoadingRows([...loadingRows, record.id]);
-                    updateBillingTeamMemberRole(billingId, record.id, key as BillingTeamRoles)
-                      .then(() => {
-                        toast.success(`User role changed to ${key}`);
-                        trackBillingTeamRoleChanged(record.email, key, billingId);
-                      })
-                      .catch(() => {
-                        toast.error("Error while changing user role");
-                      })
-                      .finally(() => {
-                        setLoadingRows(loadingRows.filter((row) => row !== record.id));
-                      });
+                    if (key !== "remove") {
+                      handleRoleChange(key as BillingTeamRoles);
+                    }
```

Comment:
We'll need to add more if else checks this way, if more actions are introduced in the future. 
* Create constants for each action item in dropdown and assign in to respective `key` in `items`
* Make use of switch case to call a function according to `key` of selected item

Reply:
```js
switch(key){
   case BILLING_TEAM_ACTIONS.MAKE_ADMIN:
        handleRoleChange();
        break;
.....
}
```

also create a function for this : `handleMemberDropdownActionClick` and pass the key to it